rialto_airflow/publish/publication.py

```diff
@@ -2,17 +2,52 @@
 from csv import DictWriter
 from pathlib import Path
 
-from sqlalchemy import select, func
-
-from rialto_airflow.database import get_session, Publication, Author, Funder
+from sqlalchemy import select, func, Boolean, Column, Integer, String
+from sqlalchemy.dialects.postgresql import insert
+from sqlalchemy.orm import declarative_base  # type: ignore
+from sqlalchemy.types import DateTime
+
+from rialto_airflow.database import (
+    RIALTO_REPORTS_DB_NAME,
+    create_schema,
+    get_session,
+    utcnow,
+    Publication,
+    Author,
+    Funder,
+)
 from rialto_airflow.utils import get_types, get_csv_path, normalize_pmid
 import rialto_airflow.publish.publication_utils as pub_utils
 
 
+ReportsSchemaBase = declarative_base()
+
+
+class PublicationAugmented(ReportsSchemaBase):  # type: ignore
```

Comment:
I wonder if we really need `Augmented` in the name here? I think this came up in discussion but we may find ourselves wanting to name these tables in such a way that they reflect what tableau visualization they are built for. But if we can get reuse of `PublicationAugmented` across visualizations that would be ideal!